src/syncAzure.proj

```diff
@@ -10,6 +10,9 @@
     <BlobNamePrefix>$(__BlobNamePrefix)</BlobNamePrefix>
     <DownloadDirectory Condition="'$(PublishTestNativeBins)' != 'true'">$(PackagesDir)AzureTransfer</DownloadDirectory>
     <DownloadDirectory Condition="'$(PublishTestNativeBins)' == 'true'">$(PackagesDir)TestNativeBins\$(RuntimeId)</DownloadDirectory>
+
+    <!-- We don't want to run sync on test native binaries for Windows, since those don't get published -->
+    <SyncingNativeOnWindows Condition="'$(PublishTestNativeBins)' == 'true' and $(RuntimeId.ToLower().StartsWith('win'))">true</SyncingNativeOnWindows>
```

Comment:
No, because we could be running on Windows but restoring for Ubuntu/OSX/Whatever. RuntimeId defines what platform we're restoring for, which is what we care about here.